react-native/react/constants/platform.native.desktop.js

```diff
@@ -19,10 +19,20 @@ const envedPathOSX = {
   prod: 'Keybase'
 }
 
+function buildWin32SocketRoot() {
+  let appdata = process.env.APPDATA
+  // Remove leading drive letter e.g. C:
+  if (/^[a-zA-Z]:/.test(appdata)) {
+    appdata = appdata.slice(2)
+  }
+  return `\\\\.\\pipe\\kbservice\\${appdata}\\Keybase`
```

Comment:
also we can use electron to get a bunch of this path stuff (maybe we should cleanup the constants file)

https://github.com/atom/electron/blob/master/docs/api/app.md#appgetpathname